### ISSUE TITLE
Missing LZ4 compression binding for Linux on AArch64/ARM64

### ISSUE THREAD

**Issue by KasaneKona** [IntellectualSites/FastAsyncWorldEdit-1.13#297]
# Bug report for FastAsyncWorldEdit 1.14.4

**FastAsyncWorldEdit Configuration Files:** 
Unable to obtain command output as server does not run. Issue does not depend on configuration files.

**FastAsyncWorldEdit Version Number:** 
Unable to obtain command output as server does not run, but using FastAsyncWorldEdit-1.15-13.jar

**Spigot/Paper Version Number:** 
`[22:56:43 INFO]: This server is running Paper version git-Paper-237 (MC: 1.14.4) (Implementing API version 1.14.4-R0.1-SNAPSHOT)`

**Description of the problem:**
When attempting to use FAWE on an Linux ARM64 system, liblz4 finds no suitable binding during loading. The server then saves and exits immediately once loading is finished.
Relevant log snippet:
```
[19:42:11] [Server thread/WARN]: java.lang.UnsupportedOperationException: Unsupported OS/arch, cannot find /linux/aarch64/liblz4-java.so. Please try building from source.
[19:42:11] [Server thread/WARN]: 	at net.jpountz.util.Native.load(Native.java:78)
[19:42:11] [Server thread/WARN]: 	at com.boydti.fawe.Fawe.setupInjector(Fawe.java:354)
[19:42:11] [Server thread/WARN]: 	at com.boydti.fawe.bukkit.FaweBukkit.<init>(FaweBukkit.java:78)
[19:42:11] [Server thread/WARN]: 	at com.sk89q.worldedit.bukkit.WorldEditPlugin.onLoad(WorldEditPlugin.java:173)
[19:42:11] [Server thread/WARN]: 	at com.sk89q.worldedit.bukkit.WorldEditPlugin.onEnable(WorldEditPlugin.java:198)
[19:42:11] [Server thread/WARN]: 	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263)
[19:42:11] [Server thread/WARN]: 	at com.sk89q.worldedit.bukkit.WorldEditPlugin.init(WorldEditPlugin.java:165)
[19:42:11] [Server thread/WARN]: 	at com.sk89q.worldedit.bukkit.WorldEditPlugin.<init>(WorldEditPlugin.java:149)
[19:42:11] [Server thread/WARN]: 	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[19:42:11] [Server thread/WARN]: 	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[19:42:11] [Server thread/WARN]: 	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[19:42:11] [Server thread/WARN]: 	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
[19:42:11] [Server thread/WARN]: 	at java.lang.Class.newInstance(Class.java:442)
[19:42:11] [Server thread/WARN]: 	at org.bukkit.plugin.java.PluginClassLoader.<init>(PluginClassLoader.java:75)
[19:42:11] [Server thread/WARN]: 	at org.bukkit.plugin.java.JavaPluginLoader.loadPlugin(JavaPluginLoader.java:134)
[19:42:11] [Server thread/WARN]: 	at org.bukkit.plugin.SimplePluginManager.loadPlugin(SimplePluginManager.java:337)
[19:42:11] [Server thread/WARN]: 	at org.bukkit.plugin.SimplePluginManager.loadPlugins(SimplePluginManager.java:257)
[19:42:11] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_14_R1.CraftServer.loadPlugins(CraftServer.java:355)
[19:42:11] [Server thread/WARN]: 	at net.minecraft.server.v1_14_R1.DedicatedServer.init(DedicatedServer.java:226)
[19:42:11] [Server thread/WARN]: 	at net.minecraft.server.v1_14_R1.MinecraftServer.run(MinecraftServer.java:876)
[19:42:11] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[19:42:11] [Server thread/INFO]: ====== LZ4 COMPRESSION BINDING NOT FOUND ======
[19:42:11] [Server thread/WARN]: java.lang.ClassCastException: java.lang.UnsupportedOperationException cannot be cast to java.lang.String
[19:42:11] [Server thread/WARN]: 	at com.boydti.fawe.Fawe.debug(Fawe.java:154)
[19:42:11] [Server thread/WARN]: 	at com.boydti.fawe.Fawe.setupInjector(Fawe.java:358)
[19:42:11] [Server thread/WARN]: 	at com.boydti.fawe.bukkit.FaweBukkit.<init>(FaweBukkit.java:78)
[19:42:11] [Server thread/WARN]: 	at com.sk89q.worldedit.bukkit.WorldEditPlugin.onLoad(WorldEditPlugin.java:173)
[19:42:11] [Server thread/WARN]: 	at com.sk89q.worldedit.bukkit.WorldEditPlugin.onEnable(WorldEditPlugin.java:198)
[19:42:11] [Server thread/WARN]: 	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263)
[19:42:11] [Server thread/WARN]: 	at com.sk89q.worldedit.bukkit.WorldEditPlugin.init(WorldEditPlugin.java:165)
[19:42:11] [Server thread/WARN]: 	at com.sk89q.worldedit.bukkit.WorldEditPlugin.<init>(WorldEditPlugin.java:149)
[19:42:11] [Server thread/WARN]: 	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[19:42:11] [Server thread/WARN]: 	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[19:42:11] [Server thread/WARN]: 	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[19:42:12] [Server thread/WARN]: 	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
[19:42:12] [Server thread/WARN]: 	at java.lang.Class.newInstance(Class.java:442)
[19:42:12] [Server thread/WARN]: 	at org.bukkit.plugin.java.PluginClassLoader.<init>(PluginClassLoader.java:75)
[19:42:12] [Server thread/WARN]: 	at org.bukkit.plugin.java.JavaPluginLoader.loadPlugin(JavaPluginLoader.java:134)
[19:42:12] [Server thread/WARN]: 	at org.bukkit.plugin.SimplePluginManager.loadPlugin(SimplePluginManager.java:337)
[19:42:12] [Server thread/WARN]: 	at org.bukkit.plugin.SimplePluginManager.loadPlugins(SimplePluginManager.java:257)
[19:42:12] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_14_R1.CraftServer.loadPlugins(CraftServer.java:355)
[19:42:12] [Server thread/WARN]: 	at net.minecraft.server.v1_14_R1.DedicatedServer.init(DedicatedServer.java:226)
[19:42:12] [Server thread/WARN]: 	at net.minecraft.server.v1_14_R1.MinecraftServer.run(MinecraftServer.java:876)
[19:42:12] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
```

**Checklist**:

- [X] I included all information required in the sections above
- [X] I made sure there are no duplicates of this report [(Use Search)](https://github.com/IntellectualSites/FastAsyncWorldEdit-1.13/issues?utf8=%E2%9C%93&q=is%3Aissue)
- [X] I made sure I am using an up-to-date version of [FastAsyncWorldEdit for 1.14.4](https://ci.athion.net/job/FastAsyncWorldEdit-pipeline/)
- [X] I made sure the bug/error is not caused by any other plugin
